index.rst

```diff
@@ -302,15 +310,15 @@ Display Name
 ^^^^^^^^^^^^
 
 A Daily display name is ``Daily [year]_[month]_[day]``, so
-``d_2021_05_11`` becomes ``Daily 2021_05_11``.  Additional components
-are handled as for weeklies.
+``d_2021_05_11`` becomes ``Daily 2021_05_11``.
+Additional components are handled as for weeklies.
 
 Semantic Version
 ^^^^^^^^^^^^^^^^
 
 The version for a daily image is ``[year].[month].[day]``.
-``d_2021_05_11`` is simply ``2021.05.11``.  Additional components go
-into the build string, as for other image types.
+``d_2021_05_11`` is simply ``2021.05.11``.
+Additional components go into the build string, as for other image types, and similarly, the RSP version (if any) is not reflected in the image's semantic version.
```

Comment:
More implementation details here, I think, since it's not clear out of context here what this means or why you're mentioning it.